new-log-viewer/src/components/Editor/MonacoInstance/theme.ts

```diff
@@ -0,0 +1,45 @@
+import * as monaco from "monaco-editor/esm/vs/editor/editor.api.js";
+
+import {THEME_NAME} from "../../../typings/config";
+import {TOKEN_NAMES} from "./typings";
+
+
+/**
+ * Sets up custom themes for the Monaco editor.
+ */
+const setupThemes = () => {
```

Comment:
Minor nit: Grammatically, this should be `setUpThemes` since `setup` is a noun, not a verb. Feel free to ignore if you think it's more confusing.

Reply:
How about `initThemes`? 

On the other hand, for consistency we should rename the other functions as well. Any objection if we do the renaming in the same PR?

Reply:
Mm, that might touch a lot of places. We could leave it as `setupXxx` for now and do the refactoring in another PR.

Reply:
Sure